mlflow/store/artifact/artifact_repo.py

```diff
@@ -150,6 +150,13 @@ def _download_file(self, remote_file_path, local_path):
         """
         pass
 
+    def delete_artifacts(self, artifact_path):
+        """
+        Delete the artifacts at the specified location.
```

Comment:
Can we specify that this supports files and directories and that directory deletion is recursive?

Reply:
Done